README.md

```diff
@@ -1,7 +1,5 @@
 # wtc-backend
-
-[![Coverage Status](https://coveralls.io/repos/github/where-to-code/wtc-backend/badge.svg?branch=master)](https://coveralls.io/github/where-to-code/wtc-backend?branch=master)
-
+[![Coverage Status](https://coveralls.io/repos/github/where-to-code/wtc-backend/badge.svg?branch=chore/set-up-C.I)](https://coveralls.io/github/where-to-code/wtc-backend?branch=chore/set-up-C.I)
 🚫 Note: All lines that start with 🚫 are instructions and should be deleted before this is posted to your portfolio. This is intended to be a guideline. Feel free to add your own flare to it.
```

Comment:
Add a free line in between the badge and the readme body.
It is preferred if the badges are at the very top of the readme, separate with a free line and let the readme title come next

Reply:
Thank Oye done